test_kcidb.py

```diff
@@ -32,6 +33,40 @@ def test_json_output_options():
                     stdout_re="\\{\n  \".*")
 
 
+def test_package_symbols():
+    """
+    Test that package modules are accessible as symbols after importing
+    the package's module (__init__.py).
+
+    Starting with the kcidb package, this test lists all modules under
+    it using pkgutil, imports the package and verifies that each listed
+    module is available as a symbol with type <class 'module'> inside it.
+    It then continues to all sub-packages recursively until everything
+    is verified.
+    """
+    package_path = kcidb.__path__[0]
+    package_name = kcidb.__name__
+    package_modules = [name for _, name, _
+                       in pkgutil.iter_modules([package_path])]
+
+    for module_name in package_modules:
+        module_fullname = f"{package_name}.{module_name}"
+        module = __import__(module_fullname, fromlist=["*"])
+        assert module_name in dir(kcidb)
+        assert isinstance(getattr(kcidb, module_name), type(module))
```

Comment:
I wonder if something like `assert getattr(kcidb, module_name, None) is module` would work instead.

Reply:
I will look into it.